cpp/cmake/thirdparty/get_jitify.cmake

```diff
@@ -18,8 +18,8 @@
 function(find_and_configure_jitify)
   rapids_cpm_find(
     jitify 2.0.0
-    GIT_REPOSITORY https://github.com/rapidsai/jitify.git
-    GIT_TAG jitify2
+    GIT_REPOSITORY https://github.com/robertmaynard/jitify.git
```

Comment:
Hey, is this official? :smiley:

Reply:
No but once https://github.com/rapidsai/jitify/pull/16 is merged we will auto pull the changes in, and I wanted this PR open to verify that it didn't break anything.

Reply:
This has been removed now that our fork of jitify has the changes we need.